client/src/__mocks__/challenge-nodes.ts

```diff
@@ -1,4 +1,17 @@
-const mockChallengeNodes = [
+interface mockChallengeNodesType {
+  fields: {
+    slug: string;
+    blockName: string;
+  };
+  id: string;
+  block: string;
+  title: string;
+  isPrivate: boolean;
+  superBlock: string;
+  dashedName: string;
+}
+
+const mockChallengeNodes: mockChallengeNodesType[] = [
```

Comment:
```suggestion
const mockChallengeNodes = [
```
TypeScript correctly infers the type from the shape of the array, so we don't need to duplicate that.